datasets/moroco/README.md

```diff
@@ -0,0 +1,169 @@
+---
+annotations_creators:
+- found
+language_creators:
+- found
+languages:
+- ro, md
```

Comment:
```suggestion
- ro-md
```
We're using the BCP-47 languages codes that includes the ISO-639-1 codes such as `ro-md` for Romanian (Republic of Moldova).

If you think that's more relevant we can also use
```suggestion
- ro
- ro-md
```